cli/src/main/resources/help/credentials.adoc

```diff
@@ -0,0 +1,76 @@
+ CREDENTIALS(1)
+==============
+:doctype: manpage
+
+
+NAME
+----
+credentials - manages keystores that contain ${infinispan.brand.name} Server credentials
+
+
+SYNOPSIS
+--------
+*credentials ls*
+
+*credentials add* 'alias'
+
+*credentials remove* 'alias'
+
+
+DESCRIPTION
+-----------
+List, create, and remove credentials inside a keystore.
+By default, commands manage the `credentials.pfx` keystore in the server configuration directory.
+
+
+SYNOPSIS
+--------
+*credentials ls*::
+Lists credential aliases stored in the keystore.
+
+Add a credential
+
+*credentials add* 'alias'::
+Adds an alias and corresponding credential to the keystore.
+
+Remove a credential
+
+*credentials remove* 'alias'::
+Deletes an alias and corresponding credential from the keystore.
+
+
+OPTIONS
+-------
+*-h, --help*::
+Prints command help.
+
+*-s, --server-root*='path-to-server-root'::
+Specifies the path to the server root directory. Defaults to `server`.
+
+*--path*='credentials.pfx'::
+Specifies the path to the credential keystore. Defaults to the server configuration directory, `server/conf`.
+
+*-p, --password*='password'::
+Specifies a password for the credential keystore.
+
+*-t, --type*='PKCS12'::
+Specifies the type of keystore that contains credentials. Supported types are `PKCS12` or `JCEKS`. Defaults to `PKCS12`.
+
+
+CREDENTIALS ADD OPTIONS
+-----------------------
+*-c, --credential*='credential'::
+Specifies the credential to store.
+
+EXAMPLES
```

Comment:
needs an extra space otherwise you get this weird formatting when doing `help credentials`:
```
  EXAMPLES
    -------- `credentials add dbpassword -c changeme -p "secret1234!"` + Creates a new default credential keystore, if does not already exist, and adds an alias of "dbpassword" for a password of "changeme". This command also sets "secret1234!" as the password for
    the credential keystore. 
```

Reply:
actually, it needs something else :/

Reply:
@pruivo That's my bad. You're right. Need two spaces between the headings or you get formatting issues. I've fixed it.